app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -196,9 +202,16 @@
       nextLevelVideo: ->
         if me.isStudent() and @course._id == utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE and !me.showHeroAndInventoryModalsToStudents()
           return utils.videoLevels[@nextLevel.original]
+      inExam: ->
+        exam = storage.load('exam', true)
+        return _.any exam.problems, (courseLevels) =>
```

Comment:
let's just return true instead of checking levels during the exam. We don't need to worry about mto students playing other things during the exam

Reply:
i don't think this takes too many time. (like only 10 loops or 20 loops). as it's simple, we can return good checking